pyproject.toml

```diff
@@ -76,6 +77,10 @@ tutorial = [
     "notebook",
 ]
 
+imas = [
```

Comment:
if I understand correctly now this is the data dictionary and should not be needed for using imas-python with TORAX.

The data dictionary may come in as a dependency of imas-python but doesn't need to be included here.